clastic/contrib/tzconvert/home.html

```diff
@@ -0,0 +1,67 @@
+<!DOCTYPE html>
```

Comment:
I think it's great to check in the complete application. However, might it be more discoverable if it were in the root `examples` directory?

`contrib` is more meant for importable/reusable elements which may have non-clastic-core dependencies*. Not sure if this is the intention in future with tzconvert, but if so, it's not mentioned in the docs.

* note to self: will probably have to add better support for this with `extra_requires` in `setup.py`

Reply:
I've hesitated between the two. I thought `contrib` was for outside contributions and `examples` were more "official". Then again, I was doubtful since `contrib` was under the `clastic` directory. Anyway, I've moved it.

Reply:
No, it's more about the dependencies than anything. :) And this one is very official! Probably the most official one yet! :)